plugins/source/mixpanel/docs/tables/mixpanel_funnel_reports.md

```diff
@@ -0,0 +1,21 @@
+# Table: mixpanel_funnel_reports
+
+The composite primary key for this table is (**funnel_id**, **date**).
+
+## Relations
+
+This table depends on [mixpanel_funnels](mixpanel_funnels.md).
+
+## Columns
+
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_source_name|String|
+|_cq_sync_time|Timestamp|
+|_cq_id|UUID|
+|_cq_parent_id|UUID|
+|project_id|Int|
```

Comment:
`project_id` should probably be part of the PK too

Reply:
Not really, funnel id is supposed to be unique (again, not documented, but looks like it)